app/controllers/external_users/claims_controller.rb

```diff
@@ -321,14 +321,14 @@ def claim_params
         :claim_id,
         :first_name,
         :last_name,
-        date_attributes_for(:date_of_birth),
+        :date_of_birth,
         :order_for_judicial_apportionment,
         :_destroy,
         representation_orders_attributes: [
           :id,
           :document,
           :maat_reference,
-          date_attributes_for(:representation_order_date),
+          :representation_order_date,
```

Comment:
This is related to the `defendants` form_step, is this required for `supporting_evidence`?